boltdesignsystem/bolt
# [6.x] DS-244 cleanup and refactor background component

## Jira

https://pegadigitalit.atlassian.net/browse/DS-244

## Summary

Shema and docs were cleaned up. Most of the work was done in [DS-823](https://github.com/boltdesignsystem/bolt/pull/2530)

## Details

- Added CSS support for gradients (Gradients can be created by adding theming class to the parent of `background-componenet`)
- Background default `opacity` option is `none`
- `shape_group` and `shape_alignment` were marked as deprecated in the docs -->  `/pattern-lab/?p=components-background-shapes-variations`
- Added docs page "Background Theme Options Variations"

## How to test

Pull the branch. Confirm that achieving solid and gradient overlay can be done by passing the theme class to the parent of the background component. Review the docs.

## Release notes

- [DS-823](https://github.com/boltdesignsystem/bolt/pull/2530)
- `shape_group` and `shape_alignment` are deprecated
- gradient overlay can be achieved by adding one of the theming class to the parent of background component
